.github/workflows/lint_build_test.yaml

```diff
@@ -46,7 +46,6 @@ jobs:
       - name: Install Dependencies
         run: |
           uv sync --frozen --all-extras --group dev
-          uv pip install --editable .
```

Comment:
?

Reply:
the sync already builds mistral-common. I added this line before you corrected the pyproject.toml to make the build because I originally thought it was necessary.

Reply:
Ok trust you that this works (test is currenly failing though I think)